### ISSUE TITLE
Train model

### ISSUE THREAD

**Issue by GabeSantini** [GabeSantini/MASDAE_IE7500_Bragi#22]
Each member will train a version of bragi.

Configurations will be focused on 2 optimizers

For each we want a version with different loss functions

Batch size, embedding dimensions and nuron counts will be modified post initial test.

Epoch should be capped at 10 for now.

After initial test, model selected will be trained by each member modifying batch, embed and nuron count.
